js/src/components/fills/MetaboxFill.js

```diff
@@ -12,11 +12,11 @@ import ReadabilityAnalysis from "../contentAnalysis/ReadabilityAnalysis";
 import SeoAnalysis from "../contentAnalysis/SeoAnalysis";
 import MetaboxCollapsible from "../MetaboxCollapsible";
 import SidebarItem from "../SidebarItem";
-import TopLevelProviders from "../TopLevelProviders";
 import AdvancedSettings from "../../containers/AdvancedSettings";
 import SocialMetadataPortal from "../portals/SocialMetadataPortal";
 import SchemaTabContainer from "../../containers/SchemaTab";
 
+/* eslint-disable complexity */
```

Comment:
what options to reduce complexity have you considered before disabling the sniff?

Reply:
Well in this case the complexity stems from each component having a condition to be rendered or not, and the only viable alternative I could think of would have been to incorporate something like an `isActive` prop and make the components themselves responsible for (not) rendering.

But all the complexity from conditionals was present before this PR too. I didn't want to go into the rabbit hole of fixing it here: my goal was to remove the duplicate Providers calls.